great_expectations/data_context/store/_store_backend.py

```diff
@@ -193,7 +193,7 @@ def _validate_key(self, key) -> None:
             for key_element in key:
                 if not isinstance(key_element, str):
                     raise TypeError(
-                        "Elements within tuples passed as keys to {} must be instances of {}, not {}".format(  # noqa: E501
+                        "Elements within tuples passed as keys to {} must be instances of {}, not {}".format(  # noqa: E501, UP032
```

Comment:
Making this change caused error because `self.__class__.__name__` had no name during a test.